test/blackbox-tests/test-cases/glob_files_rec.t

```diff
@@ -82,3 +82,24 @@ Check that generated directories are ignored
 
   $ dune build @x
 
+Check that we get a nice error message if we pass and absolute path to `glob_files_rec`
+---------------------------------------------------------------------------------------
+
+Put $PWD in a file that can be read with the %{read:...} pform, so that the underline
```

Comment:
```suggestion
Put $PWD in a file that can be read with the %{read:...} pform, so the underline
```
Here it would either be no comma and "so that..." or keep the comma and omit the "that"

Reply:
Oh cool, I feel like I'm learning so much about English grammar from your PR feedback!

Reply:
This makes me unreasonably happy!